### ISSUE TITLE
Use `nodeLinker: pnpm` for Yarn installations

### ISSUE THREAD

**Issue by jonkoops** [patternfly/patternfly-react#10548]
This issue is a follow-up of #9959.  In order to make the installation of packages more performant [`nodeLinker: pnpm`](https://yarnpkg.com/features/linkers#nodelinker-pnpm) should be configured for Yarn. This links the local node modules from a central content addressable store using symbolic links, avoiding an expensive copy operation. This has the added benefit of [catching silly bugs](https://www.kochan.io/nodejs/pnpms-strictness-helps-to-avoid-silly-bugs.html) such as [phantom dependencies](https://rushjs.io/pages/advanced/phantom_deps/#phantom-dependencies), which might lead to unexpected behaviour.

**Comment by jonkoops** [comment]
Bump

**Comment by dlabaj** [comment]
@tlabaj I think this is something worth getting in post v6.  I don't think it would be much risk to add it during what we are doing now but with the release looming I wouldn't want to introduce right now.  @jonkoops Let me know what you think.

**Comment by jonkoops** [comment]
Yes, this a development enhancement that can wait until after v6.
